src/main/java/core/basesyntax/FibonacciSequence.java

```diff
@@ -10,6 +10,15 @@
      * Результат: 21</p>
      */
     public int getFibonacciNumber(int index) {
-        return 0;
+        int result;
+
+        if (index == 0) {
+            return 0;
+        }
+        if (index == 1) {
+            return 1;
+        }
+        result = getFibonacciNumber(index - 1) + getFibonacciNumber(index - 2);
```

Comment:
Variable `result` is redundant. Consider removing it.

Reply:
done